include/tsid/formulations/inverse-dynamics-formulation-base.hpp

```diff
@@ -35,25 +36,34 @@ namespace tsid
   struct TaskLevel
   {
     EIGEN_MAKE_ALIGNED_OPERATOR_NEW
-    
-    tasks::TaskBase & task;
+
+    tasks::TaskBase &task;
```

Comment:
Please revert this "cosmetic" changes

Reply:
These lines are not coming from what I changed, I can go through all the tsid code to change all this "cosmetics" but they are not from me

Reply:
I would recommend to use clang-format with Google style if @andreadelprete is fine with it.
It can be added to the precommit action and make the PR failing if it does not respect it.
(If @nim65s has time to add it).
In line 38, they are unnecessary trailing white spaces.
It is a standard good practice nowadays.

Reply:
I am not against any style, so the one @olivier-stasse suggests is fine by me. I am just against mixing a commit of content changes with a commit of style changes. They should be two separate commits.

@Groulaxx I think these lines are coming from what you changed, otherwise they wouldn't show up in this PR. Or am I missing something? Maybe your IDE changed them automatically?

Reply:
I checked and it was my IDE, sorry for that I'll reverse it